src/minimap/Minimap.js

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useMemo } from 'react';
+import { useRef } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import Nodes from './Nodes';
+import Viewport from './Viewport';
+
+const MINIMAP_WIDTH = 200;
+const MINIMAP_HEIGHT = 150;
+const MINIMAP_STYLE = {
+  boxSizing: 'content-box',
+  width: `${MINIMAP_WIDTH}px`,
+  height: `${MINIMAP_HEIGHT}px`,
+  backgroundColor: 'rgba(240, 240, 240, 1)',
+  position: 'fixed',
```

Comment:
Why fixed?

Reply:
To keep the minimap fixed at the bottom-left position